train_redpajama.py

```diff
@@ -119,6 +118,7 @@ def train(
     train_dataloader: DataLoader,
     val_dataloader: Optional[DataLoader],
     grad_accum_steps: int,
+    devices: int,
```

Comment:
this seems to be unused :)

Reply:
Good catch, you're right. It's a remnant of when I was using it to compute token / sec / GPU in a different way